tests/unit/spring_boot_patch.py

```diff
@@ -7,8 +7,9 @@
 import tarfile
 import tempfile
 import typing
-from unittest.mock import MagicMock, patch
+from unittest.mock import MagicMock, _patch, patch
```

Comment:
Out of curiosity, why is this additional input necessary?

Reply:
This is for typing only.